README.md

```diff
@@ -1048,15 +1054,23 @@ nodeCheckInterval=2000
 ```
 It is recommended that you use --config-file to run the _percona-scheduler-admin_ script.
 
-### How to build
-----------------
+### How to build and test
+-------------------------
 
 1. Update the git submodules by executing
 
    `git submodule update --init`
 
 2. Build the scheduler submodule by running the `build_scheduler.sh`. After that we should be able to see the `pxc_scheduler_handler` binary in the base directory.
 
+3. Create admin for communication through proxysql and pxc_scheduler_handler.
+
+  Example:
+  ```sql
+  CREATE USER `admin`@`192.%` IDENTIFIED WITH 'mysql_native_password' AS 'admin' WITH ;
```

Comment:
CREATE USER `admin`@`192.%` IDENTIFIED WITH 'mysql_native_password' BY 'admin'
Please fix this

Reply:
Good observation. FIxed it.